src/main/scala/com/github/pjfanning/sourcedist/ignorelist/TarUtils.scala

```diff
@@ -6,11 +6,8 @@ import org.apache.commons.compress.compressors.gzip.GzipCompressorOutputStream
 import java.io.{BufferedOutputStream, File, FileInputStream, FileOutputStream, InputStream, OutputStream}
 
 object TarUtils {
-  def tgzFiles(tarFileName: String, filesToInclude: Seq[File], homeDir: String): Unit = {
-    val tarFile = new File(tarFileName)
-    new File(tarFileName).mkdirs()
```

Comment:
This whole code is pointless, it creates a directory at `tarFileName` and then immediately deletes it

Reply:
I'm not sure. I think the real issue is that it should be `tarFile.getParentFile.mkdirs` - the aim is/was to create the target/dist directory if it doesn't exist - when `new FileOutputStream` is used, it will fail if the directory doesn't exist.

With the change in the PR, we rely on the fact that the zip code creates target/dist anyway - but I don't like relying on the order of the code for this.

Reply:
Ah so the intention was to make the parent `dirs` folder?

If so that should sit outside of this function (also not sure it'd necessary anyways, although I do believe you have a point because `IO.zip` might be making the folder if it doesn't exist implicitly).

Reply:
Okay so being resolved in https://github.com/pjfanning/sbt-source-dist/pull/8